src/main/java/Jooh/storage/Storage.java

```diff
@@ -0,0 +1,130 @@
+package Jooh.storage;
+
+import Jooh.task.Task;
+import Jooh.task.Deadline;
+import Jooh.task.Todo;
+import Jooh.task.Event;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
```

Comment:
Maybe you could consider separating Jooh and java imports with a newline for better readability?
```suggestion
import Jooh.task.Task;
import Jooh.task.Deadline;
import Jooh.task.Todo;
import Jooh.task.Event;

import java.nio.charset.StandardCharsets;
import java.nio.file.Files;
import java.nio.file.Path;
import java.nio.file.Paths;
import java.io.IOException;
import java.util.ArrayList;
import java.util.List;
```